bridge/contracts/AllowTokens.sol

```diff
@@ -187,8 +187,8 @@ contract AllowTokens is Initializable, UpgradableOwnable, UpgradableSecondary, I
         uint256 _smallAmountConfirmations,
         uint256 _mediumAmountConfirmations,
         uint256 _largeAmountConfirmations) private {
-        require(_smallAmountConfirmations <= _mediumAmountConfirmations, "AllowTokens: small bigger than medium confirmations");
-        require(_mediumAmountConfirmations <= _largeAmountConfirmations, "AllowTokens: medium bigger than large confirmations");
+        require(_smallAmountConfirmations <= _mediumAmountConfirmations, "AllowTokens: conf small > medium");
```

Comment:
maybe keep it like before but adding 'confirmations'? like `AllowTokens: small confirmations bigger than medium confirmations`
as a reader, I guess I won't know what `conf` is

Reply:
32+ characters, do you have other suggestions until 32 characters to report?